booru-switcher.user.js

```diff
@@ -378,6 +381,7 @@ function searchByApi(host) {
     'www.trixiebooru.org': 'derpibooru',
     'derpibooru.org': 'derpibooru',
     'trixiebooru.org': 'derpibooru',
+    'manebooru.org': 'derpibooru',
```

Comment:
This line is unnecessary and will cause buggy behaviour. `searchByApi` is a Twibooru exclusive feature that currently only works with images from Derpibooru, as you'll note that all the existing domains listed all belongs to Derpibooru.

Reply:
So this is only for derpibooru translation. Okay. Removed that line from the PR in https://github.com/pawfull/Philomena-Booru-Switcher/commit/d9cfa5b215ba583b870551ef47c2b672ced3499f